### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Modules_Keyboard: When we tab from “Actions button" focus is moving to “Deny Override button" instead of “Add button".

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#87]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
**WCAG ID Violated:** WCAG2.4.3

GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #C+E-H; #.NETWebsites; #Chrome; #Edge; WCAG2.4.3

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Server and press enter
5.	Tab till Modules and press enter
6.	Validate that focus order is in a logical and meaningful order.

**Actual Result:**
Using keyboard when we tab from “Actions button” focus is moving to “Deny Override button” instead of “Add button".

**Expected Result:**
Using keyboard when we tab from “Actions button” focus should move to “Add button" instead of “Deny Override button”

**Note:**
This issue is observed in all the pages having similar controls

**User Impact:**
Users with disabilities finds its difficult in working through the application if there is inappropriate focus order using keyboard.

**Attachments:**
[Bug 87_Keyboard_Focusorder.pptx](https://github.com/Microsoft/IIS.WebManager/files/2318417/Bug.87_Keyboard_Focusorder.pptx)

**Comment by jhkimnew** [comment]
Currently this is by design. The Allow Override button is placed before the module controls. Only the problem is that we used "pull-right" such as <override-mode class="pull-right"> which caused the button is displayed in the same line of the first module control. This can be fixed in the future.

**Comment by Roopesh-T** [comment]
**GitTag**: #A11ySev2;

**Comment by Yashi1201** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will be removed.
